### PR TITLE
Add 1.46 role

### PR DESCRIPTION
**Pull request by aljesusg**
1.46 will be replaced by 1.48 

Do not merge

### CODE REVIEW

File: manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml [hunk 1]
```diff
@@ -103,7 +103,7 @@ spec:
 
     For quick descriptions of all the settings you can configure in the Kiali
     Custom Resource (CR), see the file
-    [kiali_cr.yaml](https://github.com/kiali/kiali-operator/blob/v1.36/deploy/kiali/kiali_cr.yaml)
+    [kiali_cr.yaml](https://github.com/kiali/kiali-operator/blob/v1.46/deploy/kiali/kiali_cr.yaml)
```
Comment (jmazzitelli) [inline manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml:106]: This section (lines 104-106) is different now. 1.46 branch won't have the kiali_cr yaml - so you need to change this to look like what master looks like now: https://github.com/kiali/kiali-operator/blob/master/manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml#L104-L106

```
    For quick descriptions of all the settings you can configure in the Kiali
    Custom Resource (CR), see
    [the kiali.io docs](https://kiali.io/docs/configuration/kialis.kiali.io/).
```

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
Aside from the help text that needs to change in the CSV, this looks good.

But can you confirm that the `roles/v1.46` is simply a copy of the current `roles/default`? That's what it should be (I just want to make sure you didn't copy `roles/v1.36` - the new version should be a copy of the `roles/default` directory).

I also +1 the changes you made to add the new 1.46 to the default supported images. That is all correct.

**Comment by aljesusg** [comment]
> Aside from the help text that needs to change in the CSV, this looks good.
> 
> 
> 
> But can you confirm that the `roles/v1.46` is simply a copy of the current `roles/default`? That's what it should be (I just want to make sure you didn't copy `roles/v1.36` - the new version should be a copy of the `roles/default` directory).
> 
> 
> 
> I also +1 the changes you made to add the new 1.46 to the default supported images. That is all correct.

I am 100% sure that I copied the 1.36 and not the default 🤣 let me change it 😊

**Review by jmazzitelli — APPROVED** [review]

**Comment by aljesusg** [comment]
I think @jmazzitelli that if we are going to update this later to the version 1.48 we can avoid this merge and only do the PR to the branch 1.46 https://github.com/kiali/kiali-operator/pull/488 . What do you think?

**Comment by jmazzitelli** [comment]
> only do the PR to the branch 1.46 #488 . What do you think?

yes, that's a good idea.

**Comment by aljesusg** [comment]
Closed in favor https://github.com/kiali/kiali-operator/pull/488

**Pull request closed without merging**